ocaml-lsp-server/docs/ocamllsp/config.md

```diff
@@ -42,5 +42,12 @@ interface config {
   * @since 1.18
   */
   syntaxDocumentation: { enable : boolean }
+
+  /**
+  * Enable/Disable Merlin Jump code actions
+  * @default true
+  * @since 1.19
+  */
+  merlinJumpsCodeAction: { enable : boolean }
```

Comment:
This doesn't match the actual name of the config option: `merlinJumpCodeActions`

Reply:
thanks for catching that